packages/i18n/load-i18n.js

```diff
@@ -46,7 +46,12 @@ export default function loadI18n(lang) {
   const localeDataPromises = [reactIntlChunkImport, momentChunkImport];
 
   return Promise.all(localeDataPromises).then(response => {
-    addLocaleData([...response[0].default]);
-    return getLocalizedStringsChunkImport(lang).then(result => result.default);
+    // Prefer loading `default` (for ESM bundles) and
+    // fall back to normal import (for CJS bundles).
+    const loadedData = response[0].default || response[0];
+    addLocaleData([...loadedData]);
```

Comment:
Why do we spread and not just pass loadedData?

Reply:
I think it was copy-pasted from the l10n implementation where we spread more than one. We can make it simpler sure 😊